nbs/03_Data_Exploration.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<p>Try having a consistent naming convention for the variable names. Ref: https://www.python.org/dev/peps/pep-0008/#naming-conventions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>